tests/acceptance/reminder-list-test.js

```diff
@@ -97,15 +98,53 @@ test('user message disappears if a user adds a new reminder', function(assert) {
   visit('/');
   click('.spec-add-new');
 
-  andThen(function() {
-    fillIn('.title-input', 'walk dog');
-    fillIn('.date-input', 'today');
-    fillIn('.notes-input', 'Fido is a good dog');
-    click('.submit-new-btn');
-  })
+  fillIn('.title-input', 'walk dog');
+  fillIn('.date-input', 'today');
+  fillIn('.notes-input', 'Fido is a good dog');
+  click('.submit-new-btn');
 
   andThen(function() {
     assert.equal(find('.user-message').text().trim(), '')
     assert.equal(find('.reminder').length, 1)
   })
 })
+
+test('user should be able to edit the reminder', function(assert) {
+  visit('/');
+  click('.spec-add-new');
+
+  andThen(function(){
+    assert.equal(currentURL(),'/reminders/new');
+  })
+
+  fillIn('.title-input', 'Taylors mom');
+  fillIn('.date-input', 'always');
+  fillIn('.notes-input', 'Has got it going on');
+  click('.submit-new-btn');
+  click('.spec-reminder-title:first');
+
+  andThen(function(){
+    assert.equal(currentURL(),'/reminders/1');
+  })
+
+  click('.edit-reminder-btn')
+
+  andThen(function(){
+    assert.equal(currentURL(),'/reminders/1/edit');
+  })
+
+  fillIn('.edit-title', 'Brennas mom');
+  fillIn('.edit-date', 'always');
+  fillIn('.edit-notes', 'Has got it going on, too');
+  click('.save-reminder-btn');
+
+  andThen(function(){
+    assert.equal(currentURL(), '/reminders');
```

Comment:
I'd prefer it if the user was redirected to the particular reminder page they just edited. Let's update this test to reflect that and add a `.then()` and `transitionTo()` method in the save action. [This guy](https://discuss.emberjs.com/t/how-to-redirect-to-an-object-show-route-after-its-creation/9645) talks about it in his  blog post.